mongo/check.py

```diff
@@ -642,7 +642,9 @@ def total_seconds(td):
 
         if ssl_params:
             username_uri = u"{}@".format(urllib.quote(username))
-            clean_server_name = clean_server_name.replace(username_uri, "")
+
+            if username_uri:
```

Comment:
At this point `username_uri != None` because it's assigned above, so I'm not sure this condition is necessary. Maybe you meant: 

```
if username:
    username_uri = u"{}@".format(urllib.quote(username))
    clean_server_name = clean_server_name.replace(username_uri, "")
```